io/plugins/eu.esdihumboldt.hale.io.shp/src/eu/esdihumboldt/hale/io/shp/writer/ShapefileInstanceWriter.java

```diff
@@ -724,4 +730,29 @@ private List<String> writeToFile(
 		return filesWritten;
 	}
 
+	/**
+	 * 
```

Comment:
I meant that you can move the comment : `Create the CPG file starting from the Shapefile names` here to this newly added method as it belongs here.